curp/src/server/mod.rs

```diff
@@ -189,6 +189,13 @@ impl<C: Command + 'static> Rpc<C> {
             .await?;
         Ok(())
     }
+
+    /// Get a subscriber for leader changes
+    #[inline]
+    #[must_use]
+    pub fn leader_rx(&self) -> broadcast::Receiver<Option<ServerId>> {
```

Comment:
This subscriber is never used in this change. Why?

Reply:
I guess @themanforfree will add the usage in his fork in the next PR.

Reply:
I think we can add tests for this, like you did for client in this PR

Reply:
Sure. Added.